docs/sdks/create-api-key.md

```diff
@@ -29,8 +29,23 @@ client = C8Client(protocol='https', host='gdn.paas.macrometa.io', port=443,
                     email='nemo@nautilus.com', password='xxxxx',
                     geofabric='_system')
 
-# Create an API key
-print("Create API Key: ", client.create_api_key('id1'))
+# Check wether the chosen_api_key exists or not, comparing it to the list elements
```

Comment:
```suggestion
# Checks if `chosen_api_key` exists
```

Reply:
Same comment as on other PR, not sure what we mean by "list elements" but we might want to add that part back in.

Reply:
Sorry, I missed adding a few lines of code. Fixed.

A list of all API key IDs a user has is saved in a variable which is used to make a comparison check later.

Reply:
I have updated the comment to improve semmantics.